pkg/virt-launcher/virtwrap/util/libvirt_helper.go

```diff
@@ -491,9 +491,13 @@ func copyFile(from, to string) error {
 }
 
 func (l LibvirtWraper) SetupLibvirt() (err error) {
-	runtimeQemuConfPath := path.Join(libvirtRuntimePath, "qemu.conf")
-	if err := copyFile(qemuConfPath, runtimeQemuConfPath); err != nil {
-		return err
+	runtimeQemuConfPath := "/etc/libvirt/qemu.conf"
```

Comment:
I would have this as a constant in the beginning of the file, next to the other constants you defined.
Something like:
```
const (
	qemuRootConfPath       = "/etc/libvirt/qemu.conf"
	qemuNonRootConfPath = "/etc/libvirt/qemu.conf"
	libvirdConfPath    = "/etc/libvirt/libvirtd.conf"
	libvirtRuntimePath = "/var/run/libvirt"
)
```

Reply:
I am not sure that will compile.

Reply:
You're right. This will:
`qemuNonRootConfPath = libvirtRuntimePath + "/qemu.conf")`

Reply:
It probably won't compile since it's not a constant expression.

However, I don't see the point of using the function `filepaths.Join()` here, we don't care about being portable about the directory separator.

Just using the constant exepression `qemuNonRootConfPath = libvirtRuntimePath + "/qemu.conf"` should work.